content/blog/2021-05-14-inventing-the-service-trait.md

```diff
@@ -0,0 +1,955 @@
+---
+date: "2021-05-17"
+title: "Inventing the Service trait"
+description: "May 17, 2021"
+---
+
```

Comment:
We should add a note here either at the top or bottom saying this is a guide that exists within the tower repo and link to it to avoid possible confusion.

Reply:
Once this is published I'll replace the one in tower with a link to this.